scipy/stats/morestats.py

```diff
@@ -960,7 +960,9 @@ def boxcox(x, lmbda=None, alpha=None):
         If ``alpha`` is not None, return the ``100 * (1-alpha)%`` confidence
         interval for `lmbda` as the third output argument.
         Must be between 0.0 and 1.0.
-
+    bounds : sequence, optional
+       Optional bounds for `lmbda`. Must have two items corresponding to the
+       optimization bounds. For more details, see :func:`boxcox_normmax`.
```

Comment:
Consider "Lower and upper bounds on `lmbda`" to be more explicit.

Go ahead and copy the details in here. If you don't want to copy-paste, you can probably define a variable to be substituted in both places as in `_distn_infrastructure.py`, but that may be more trouble than it's worth for two places.